articles/openstack/ostack-how-install-rhui.md

```diff
@@ -0,0 +1,100 @@
+---
+title: How to install Red Hat Update Infrastructure on an existing OpenStack instance | UKCloud Ltd
+description: Shows you how to update your existing OpenStack Instances to target UKCloud's approved Red Hat Update Infrastructure (RHUI)
+services: openstack
+author: Sue Highmoor
+toc_rootlink: How To
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Install Red Hat Update Infrastructure on an existing OpenStack instance
+toc_fullpath: How To/ostack-how-install-rhui.md
+toc_mdlink: ostack-how-install-rhui.md
+---
+
+# How to install Red Hat Update Infrastructure on an existing OpenStack instance
+
+## Overview
+
+This article provides advice on how to update your existing Red Hat virtual machines (VMs) to target UKCloud's approved Red Hat Update Infrastructure (RHUI).
+
+As of July 2015, UKCloud implemented an RHUI to provide automatic updates to our Red Hat customers on our Assured OFFICIAL and Elevated OFFICIAL security domains. This provides benefits such as the reliable availability of patch updates and Red Hat approved OS templates.
+
+This update service replaces the previous repositories, which are no longer updated and were retired on the 31 August 2015.
+
+## Prerequisites
+
+There are two pre-requisites for performing this operation:
+
+- The host can resolve to the RHUI DNS records (contact Customer Support for the IP addresses and domain names for these records). You can achieve this by configuring an A record on your local DNS, or configuring an `/etc/hosts` file with the appropriate information. For Internet connectivity, the required domain name is in public DNS.
+
+- All hosts using the service must be able to access the local RHUA server on port 443 (HTTPS). Ensure that the configured Security Groups allow outbound connectivity on port 443. If you have any questions on this, contact UKCloud Support.
+
+## Installation
+ 
+You can find the installer RPM files in an ISO image. Before this can be mounted to an existing Instance, you will need to create a volume from the image.
+
+1. In the Horizon Web UI, locate the image `UKCloud-RHUI-Standard-Assured.iso`, click the action button and select `Create Volume`:
```

Comment:
Sorry, just use backticks for the file name (UKCloud-RHUI-Standard-Assured.iso). For the UI field labels (e.g., Create Volume, Attach Volume, etc.) use bold (**)